src/docs/development/platform-integration/web.md

```diff
@@ -25,7 +25,9 @@ No.
 
 ### Can I use Flutter plugins?
 
-Not yet.
+Yes, several plugins have web support. 
+Find an updated list of plugins on [pub.dev][] using the web filter.
```

Comment:
Can you provide a direct link to this filtered list?  Here's the link:  https://pub.dev/dart/packages?runtime=web

Reply:
@sfshaza re: the link you provided: aren't those dart packages?

Reply:
i think packages and plugins are mixed... i went on pub.dev and filtered by flutter and web to get this list : https://pub.dev/flutter/packages?platform=web 

which seems accurate to me as plugins

Reply:
Great!